fourmolu.yaml

```diff
@@ -11,4 +11,5 @@ let-style: auto
 in-style: right-align
 unicode: never
 respectful: true
-fixities: []
\ No newline at end of file
+single-constraint-parens: never
```

Comment:
This won't have an effect until after we upgrade to `0.12.0.0`, and it will cause another mass change. I have no problem with this, just noting.

Reply:
In `0.10.1.0` it defaults to `never` but in `0.12.0.0` it's `always` so doing this just means our code won't change once we upgrade 😄